app/controllers/article_controller.rb

```diff
@@ -186,4 +186,8 @@ def setup_eds_session(session)
       debug: Settings.EDS_DEBUG
     ).session_token
   end
+
+  def has_search_parameters?
```

Comment:
@jkeck it's working! weirdly enough, I had to add it towards the top of the controller. Adding it below the method didn't work for some reason 🤔